### ISSUE TITLE
role="group" without aria-labelledby

### ISSUE THREAD

**Issue by shabana-ali** [hmrc/accessibility#49]
Initially identified this on an audit https://hmrcdigital.slack.com/archives/G01EQBX3H53/p1605632709006300

Guidance stating `role="group"` can be labelled using `aria-labelledby` https://www.w3.org/TR/WCAG20-TECHS/ARIA17
After discussions with Adam, concluded it is not a WCAG 1.3.1: Info and Relationships fail if `aria-labelledby` not present.

However, `role="group"` is a viable alternative when it is difficult to employ the fieldset-legend technique - both are not required as per the design system example: https://design-system.service.gov.uk/components/date-input/default/index.html

Need to investigate / discuss and raise with GDS if a concern.

**Comment by ashfaqhussain357** [comment]
Issue fixed.
